tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -236,11 +252,12 @@ def traverse_node(node):
         traverse_node(root_node)
         return root_node
 
-    def parse_events(self, events):
+    def parse_events(self, events, context_data):
 
         def parse_event(event, corrid_to_device, corrid_to_runtime, externalid_to_runtime, tid2list, tid2zero_rt_list):
             corrid = event.args.get("correlation", None)
-            input_shape = event.args.get("Input dims", None)
```

Comment:
we can move these logic to be @property of TraceEvent class.

Reply:
thanks, done